task_handler_layer/task_gate/src/gate_detection.cpp

```diff
@@ -268,10 +290,11 @@ int main(int argc, char *argv[])
           p = 32;
       }
       if (p == 32)
-        ROS_INFO("%s: PAUSED\n", ros::this_node::getName().c_str());
+        printf("paused\n");
```

Comment:
this code was not  updated in nuc hence lots of these problems I guess, anyways done.